libi2pd/Config.cpp

```diff
@@ -281,7 +284,7 @@ namespace config {
 
 		options_description meshnets("Meshnet transports options");
 		meshnets.add_options()
-			("meshnets.yggdrasil", bool_switch()->default_value(false),              "Support transports through the Yggdrasil (default: false)")
+			("meshnets.yggdrasil", bool_switch()->default_value(false),              "Support transports through the Yggdrasil (deafult: false)")
```

Comment:
Looks like a typo @orignal

Reply:
https://github.com/PurpleI2P/i2pd/commit/6971b1e9da863a7260b145dde29f5961c74feaf0

Reply:
Out of curiosity, how does development work on this repo? Because changes are being pushed to a branch with a merged PR. Do you open a new PR under the same branch later?

Reply:
All changes done in `openssl` branch. Sometimes @orignal merge changes from `openssl` to `master`, when changes which can break something is pushed. So, master is in many cases is stable code.

Or what you mean?

Reply:
Thanks for clarifying